src/database/queries/editcohort.js

```diff
@@ -0,0 +1,17 @@
+const dbconnection = require('../db_connection');
+
+const editCohorts = (name, description, githublink, imgURl, cohortID) => new Promise((resolve, reject) => {
```

Comment:
Don't pass all of these as parameters, pass them to this function as one object and destructure them inside the `editCohorts`.
i.e.
```js
// Callee file
editCohorts({ name, description, githubLink, imgUrl, cohortId });

// editCohort.js
const editCohorts = (cohortData) => {
  const { name, description, githublink, imgURl, cohortID } = cohortData;
...
```
Too, rename your `githublink` to `githubLink`, your `imgURl` to `imgUrl` or `iMgUrL`, your `cohortID` to `cohortId`, and your `editcohort.js` to `editCohorts.js`